driver-core/src/main/java/com/datastax/driver/core/CBUtil.java

```diff
@@ -117,10 +118,47 @@ public static void writeBytes(byte[] bytes, ByteBuf cb) {
         cb.writeBytes(bytes);
     }
 
+    public static void writeBytes(ByteBuffer bytes, ByteBuf cb) {
```

Comment:
Is it said somewhere that values are never null? Based on the protocol spec, I assumed they could (hence my other comment on readBytesMap).